q0.ui

```diff
@@ -266,22 +269,36 @@
         </layout>
        </widget>
       </item>
-      <item row="2" column="3">
-       <widget class="QGroupBox" name="groupBox_8">
+      <item row="2" column="1" colspan="2">
+       <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
-         <string>Heater</string>
+         <string>JT</string>
         </property>
-        <layout class="QGridLayout" name="gridLayout_10">
-         <item row="1" column="1" colspan="2">
-          <widget class="PyDMSpinbox" name="heater_setpoint_spinbox">
+        <layout class="QGridLayout" name="gridLayout_5">
+         <item row="1" column="0">
+          <widget class="QLabel" name="label_6">
+           <property name="text">
+            <string>Setpoint:</string>
+           </property>
+          </widget>
+         </item>
+         <item row="0" column="2">
+          <widget class="PyDMLabel" name="jt_mode_label">
            <property name="toolTip">
             <string/>
            </property>
-           <property name="maximum">
-            <double>160.000000000000000</double>
+           <property name="alarmSensitiveContent" stdset="0">
+            <bool>true</bool>
            </property>
-           <property name="singleStep">
-            <double>10.000000000000000</double>
+          </widget>
+         </item>
+         <item row="1" column="1">
+          <widget class="PyDMSpinbox" name="jt_setpoint_spinbox">
+           <property name="toolTip">
+            <string/>
+           </property>
+           <property name="maximum">
+            <double>75.000000000000000</double>
```

Comment:
Should this also be 160? or stay 75 here?

Reply:
Different system! The JT doesn't actually go below 5% or above 70%

Reply:
so should the max be 70%?